assets/src/styles/blocks/Articles/ArticlesEditorStyle.scss

```diff
@@ -21,9 +21,9 @@
     &:hover,
     &:focus,
     &:active {
-      background-color: transparentize($white, .7);
-      border-color: $dark-blue;
-      color: $dark-blue;
+      background-color: rgba($white, .7);
```

Comment:
Don't we want to replace `$white` with `var(--white)` here?

Reply:
Nope! We've replaced the `transparentize` (SASS function) to `rgba` and to use native implementations. `rgba` does not work with `CSS Variables`, only with `hexa` colors. [That's why we also added `SASS Variables` within the tokens](https://github.com/greenpeace/planet4-design-tokens/pull/8).